core_backend/deployment/bootstrap_demo.sh

```diff
@@ -56,8 +56,7 @@ export PROMETHEUS_MULTIPROC_DIR="/tmp"
 
 DOMAIN="aaq-demo.idinsight.io"
 
-# Resolve Qdrant host
-SERVICE_RECORD=$(dig +short SRV ${QDRANT_HOST})
+
```

Comment:
Had a huddle with Carlos and it makes sense to just delete the stuff now instead of in a separate PR - the qdrant resources won't be getting used once this is merged in so might as well remove them now.